scripts/crond/back-nightly

```diff
@@ -7,7 +7,7 @@
 
 
 IMAGE_PROCESS_FLAG=1
-PIPELINE_UPDATE_FLAG=1
+PIPELINE_UPDATE_FLAG=0
```

Comment:
What's the reason for this change?

I think that with this things like import_mr_sessions, export_measures and update_csv_summaries won't run

Reply:
I have to disable the cronjobs to begin generating the release. You said to put it in this PR.
Also, there are some uncommited changes in sibis-software's ncanda-data-integration which are going to prevent pulling this change after it's been merged into master. Do you know where those came from? @jimklo

Reply:
lets talk at 1pm aboutit